proof-producer/bin/proof-producer/src/main.cpp

```diff
@@ -72,7 +72,8 @@ int run_prover(const nil::proof_generator::ProverOptions& prover_options) {
                     }
                     break;
                 case nil::proof_generator::detail::ProverStage::ASSIGNMENT:
-                    prover_result = prover.setup_prover() && prover.fill_assignment_table(prover_options.trace_base_path);
+                    prover_result = prover.setup_prover(prover_options.circuits_limits) &&
+                    prover.fill_assignment_table(prover_options.trace_base_path, AssignerOptions(false, prover_options.circuits_limits));
```

Comment:
Let's use designated initializers instead of a constructor (since it's trivial). They do really improve readability:
```suggestion
                    prover.fill_assignment_table(prover_options.trace_base_path, AssignerOptions{
                    .ignore_index_mismatch = false,
                    .circuits_limits = prover_options.circuits_limits});
```

Reply:
done